akka-http-tests/src/test/scala/akka/http/scaladsl/unmarshalling/MultipartUnmarshallersSpec.scala

```diff
@@ -162,6 +180,25 @@ trait MultipartUnmarshallersSpec extends AnyFreeSpec with Matchers with BeforeAn
             Multipart.General.BodyPart.Strict(HttpEntity(ContentTypes.`text/plain(UTF-8)`, "first part, implicitly typed")),
             Multipart.General.BodyPart.Strict(HttpEntity(`application/octet-stream`, ByteString("second part, explicitly typed"))))
       }
+      "a full example (Default) containing a CR in the body part" in {
```

Comment:
This one even didn't fail before, probably because even before a `\r` was only considered in probing if it was part of the first received chunk.